Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLEnergyTransfer.py

```diff
@@ -7,18 +7,29 @@
 from __future__ import (absolute_import, division, print_function)
 
 import os
+import math
 import numpy as np
 from mantid import config, mtd, logger
-from mantid.kernel import StringListValidator, Direction
+from mantid.dataobjects import TableWorkspace
+from mantid.kernel import StringListValidator, Direction, FloatBoundedValidator, \
+    FloatArrayProperty, FloatArrayLengthValidator, IntBoundedValidator
 from mantid.api import PythonAlgorithm, MultipleFileProperty, FileProperty, \
-    WorkspaceGroupProperty, FileAction, Progress
-from mantid.simpleapi import *  # noqa
+    WorkspaceGroupProperty, FileAction, Progress, WorkspaceProperty, PropertyMode
+from mantid.simpleapi import *
+
+
+N_TUBES = 16
```

Comment:
For the moment this algorithm supports one instrument, which is not going to change soon. I think it is fine to have this hardcoded for the moment, ultimately this can be put in the instrument parameters.

Reply:
:+1: